index.js

```diff
@@ -0,0 +1,90 @@
+// Initializing all elements constants
+const temperateField = document.querySelector(".weather1");
+const cityField = document.querySelector(".weather2 p");
+const dateField = document.querySelector(".weather2 span");
+const emojiField = document.querySelector(".weather3 img");
+const weatherField = document.querySelector(".weather3 span");
+const searchField = document.querySelector(".searchField");
+const form = document.querySelector("form");
+
+// Function to fetch Data from Weather API
+const fetchData = async (target) => {
+  try {
+    const url = `https://api.weatherapi.com/v1/current.json?key=5b27a6ef3547402582e62007222306&q=${target}`;
```

Comment:
Generally, such API Url's should be kept in separate file `constants.js`, put this url there and import from `constants.js` to here